static/admin/config.yml

```diff
@@ -13,20 +13,10 @@ collections:
     delete: false
     files:
       - file: "src/data/imprint/imprint.json"
-        name: "imprint"
+        name: "theImprint"
```

Comment:
Why this change?

Reply:
The collection is named imprint 

https://github.com/hardforkio/homepage-new/blob/244d6d265575bd002c51a8a121bf05cfc1d4f4fd/static/admin/config.yml#L10-L11

I think this the name (ID) of the first (and only) entry of the collection
https://github.com/hardforkio/homepage-new/blob/244d6d265575bd002c51a8a121bf05cfc1d4f4fd/static/admin/config.yml#L14-L16